src/acquire-device-properties/device/props/storage.h

```diff
@@ -31,6 +31,16 @@ extern "C"
                 uint32_t width, height, planes;
             } tile;
         } chunking;
+
+        /// Parameters for multiscale.
+        /// `max_layer` is the maximum number of layers to generate, where 0 is
+        /// the original resolution, 1 is the first downscale, etc. -1 means to
+        /// generate as many layers as will downsample the image to the tile
+        /// shape.
+        struct storage_properties_multiscale_s
+        {
+            int16_t max_layer;
+        } multiscale;

```

Comment:
I really only see an enable/disable option being needed here. No need for the user to specify/compute the `max_layer`.

I don't see any real utility in allowing the user to specify the number of layers, and it's pretty easy to set wrong so I prefer striking that parameter.

When enabled, the number of scales to compute should just be determined the desired chunk size - the image is down-sampled until there is less-than-or-equal to a chunk at the coarsest scale.